### PR TITLE
Create : Simplified Grid System created

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Created basic 12 column grid system to make responsive web design

### CODE REVIEW

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+.row div{
+    background-color: #7fffd4;
+    padding: 2rem 5rem;
+    text-align: center;
+}
+
+.col-1{
+    grid-column: span 1;
+
```
Comment (sainath7878) [inline components/grid.css:19]: remove extra lines

File: documentation/docs.html [hunk 2]
```diff
@@ -447,6 +451,46 @@ <h3 class="product-owner">By Apple</h3>
                     style="height: 481px; width: 100%; border:0; transform: scale(1); overflow:hidden; background: #151718;"
                     sandbox="allow-scripts allow-same-origin">
                 </iframe>
+            </div>
+        </section>
+
+        <!-- Grid -- component -->
+        <section id="drid" class="component-container">
```
Comment (sainath7878) [inline documentation/docs.html]: type error I guess. id should grid

Comment (sainath7878) [reply documentation/docs.html]: Grid is not accessible from side navbar

**Review by Shubhamkr2610 — APPROVED** [review]
Looks Good to me. You can merge

**Pull request merged**